core/src/test/resources/reference.conf

```diff
@@ -98,6 +98,14 @@ dataRepo {
   enabled = true
 }
 
+resourceBuffer {
+  projectPool {
+    regular = "dev-project-regular-v1"
+    servicePerimeter = "dev-project-serviceperimeter-v1"
+  }
+  url = "https://rbs.dsde-dev.broadinstitute.orgLOCAL"
```

Comment:
Should this have a `:` between `.org` and `LOCAL`?

Reply:
I think this file is used for unit tests